src/frontend/src/flows/manage/index.ts

```diff
@@ -130,50 +152,57 @@ const style = () => html`<style>
 // recovery devices.
 const pageContent = (userNumber: bigint, devices: DeviceData[]) => html`
   ${style()}
-  <div class="container">
+  <section class="container">
```

Comment:
Is this purposeful, having the `container` div be a section sometimes, and sometimes an article as in [here](https://github.com/dfinity/internet-identity/blob/51968c98bb6c68e20bbecc68e9af484edff9840f/src/frontend/src/flows/addDevice/manage/index.ts#L45)? Is it because of different purpose in the flows? 
I initially thought it would make sense to keep as one consistent type.

Reply:
Yes it is. Its to transport the meaning of the information it contains

> Represents a generic document or application section. In this context, a section is a thematic grouping of content, typically with a header, possibly with a footer. Examples include chapters in a book, the various tabbed pages in a tabbed dialog box, or the numbered sections of a thesis. A web site's home page could be split into sections for an introduction, news items, contact information. [Section HTML5 Doctor](http://html5doctor.com/element-index/#s)

The article is more of a single widget, that could be anywhere else on the website. I thought it was more appropriate in this case. But I think its not a clear cut in this case. Since it contains two things that are quite different semantically I thought it would be nice to group them loosely.

I hope this answers your question. Don't hesitate to ping me if not.